### ISSUE TITLE
License file?

### ISSUE THREAD

**Issue by hrieke** [jsonlines/jsonfilter#3]
Hi would be it possible to have a better license listed with the project?
While the package.json file says the license is BSD, it does not spell out which BSD license is in effect (BSD zero, 1, 2, 3, 4, or 5?)
Thanks

**Comment by brettz9** [comment]
Yes, it'd be appreciated if the desired SPDX identifier could be used: https://spdx.org/licenses/ such as `BSD-2-Clause`.

Also, the version of JSONStream on which `jsonfilter` is depending, 0.8.4, does not have any `license` meta-data in its `package.json`, so `jsonfilter` bumping that dependency would really be needed for full license clarity by users of this package. (Though JSONStream is now archived, its more recent version did at least have valid `license` meta-data.)
